Readme upgrade

marcintreder opened the pull request:
Changes:
- link to Merge in the first section
- update of 'add new components' section
- update of JSX Preset section (JSON preset removed)
- added CI integration section

marcintreder committed:
Update config.yml

marcintreder committed:
Update config.yml

Commit:
Update of presets part of readme

marcintreder committed:
Update README.md

marcintreder committed:
Update README.md

michalstocki commented on .circleci/config.yml:
```diff
@@ -1,22 +1,13 @@
           name: "Install dependencies"
           command: npm install
-      - run:
-          name: "Run tests"
```
Why did you removed it?

marcintreder replied:
@michalstocki actually I was just meaning to change the build/docker part of the config. I don't think it makes sense to build this repository with our image and require credential to docker. This repo is meant to be forked by users

michalstocki commented on README.md:
```diff
@@ -57,77 +59,240 @@ Adding components to Merge is no different than creating normal React.js compone
-      }
+Every component has to be referenced in UXPin Merge Config file [`./uxpin.config.js`](https://wikiuxpin.atlassian.net/wiki/spaces/MA/pages/665714725/UXPin+Merge+Config+File).
+
+## Authoring JSX Presets
```
I think it's a bad idea to duplicate here the documentation of authoring the presets on CI configuration. IMHO we should put here only a link to the wiki documentation, otherwise it may be really hard to maintain the same documentation in different places.

michalstocki commented on README.md:
```diff
@@ -57,77 +59,240 @@ Adding components to Merge is no different than creating normal React.js compone
 * You have to add the component to uxpin.config.js file
 * You have to prepare presets for every component (temporary restrictions to be replaced by jsx presets)
```
This point should be corrected.
```suggestion
 * You have to prepare a default preset for every component
```

michalstocki commented on README.md:
```diff
@@ -11,6 +11,8 @@ To make the first steps in the world of Merge easy, we've prepared this boilerpl
 
+To learn more about Merge – visit [our wiki](https://wikiuxpin.atlassian.net/wiki/spaces/MA/overview)
+
 ## Installation
```
I think the `## Tests` section is in a wrong place – I'd put it at the bottom of the readme, or at least under "Adding new components", as it's a detail important for repository maintainers, not a users experimenting with the technology.

michalstocki committed:
Update README.md

Piece of the old preset stayed in the readme file. This commit removes it.

Co-Authored-By: marcintreder <marcintreder@gmail.com>

marcintreder committed:
Updating Circle CI configuration to add ESLint and Tests

marcintreder committed:
Relocating Tests info to the bottom in Readme

marcintreder committed:
Removed JSX guide from Readme

Link to WIKI was provided instead of JSX section. Hopefully this is going to be easier to maintain.
